composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^7.1 || ^8.0",
         "psr/http-factory": "^1.0",
-        "phpunit/phpunit": "^6.5 || ^7.0 || ^8.0 || ^9.0"
+        "phpunit/phpunit": "^6.5 || ^7.0 || ^8.0 || ^9.0 || ^10.0"
```

Comment:
Oh. That's disappointing.

Reply:
The branch I linked in the OP is a cleanup that supports PHP 7.4+ - you can take it and do whatever you want with it. Alternatively, you can have the PHP 8.1+ version that I've [included in my library's tests](https://github.com/chillerlan/php-httpinterface/tree/main/src/Psr17) for the time being because this package breaks CI in its current form.

Reply:
> We can't do that without dropping PHP 7.1 and 7.2 support.

Can't those using older PHP versions continue using older release of this package?